Task1/Program.cs

```diff
@@ -10,7 +10,268 @@ class Program
+                {
+                    count++;
+                }
+                a = a / 10;
+            }
+            Console.WriteLine($" The number of even digits in the number {b} is {count}");
+        }
+
+        public static void MeanOfNumbers()
+        {
+            Console.WriteLine("Enter two numbers A and B( A<=B)");
+            int number1 = Convert.ToInt32(Console.ReadLine());
+            int number2 = Convert.ToInt32(Console.ReadLine());
+            int sum = 0;
+            int count = 0;
+            for (int i = number1; i <= number2; i++)
+            {
+                sum += i;
+                count++;
+            }
+            double average = (double)sum / count;
+            Console.WriteLine($"Average of numbers between {number1} and {number2} ( inclusive) is {average}");
+        }
+
+        public static void DaysOfRunning()
+        {
+            double distance = 10;
+            double sum = 10;
+            int days = 1;
+            while (sum <= 100)
+            {
+                distance = distance * 1.1;
+                sum = sum + distance;
+                days++;
+            }
+            Console.WriteLine($"Skier ran {sum} for {days} days");
+        }
+
+        public static void Square(int N)
+        {
+            Console.WriteLine($"Squares of numbers less than {N} is: ");
+            int i = 1;
+            while (i*i<N)
+            {
+                Console.Write(i*i +" ");
+                i++;
+            }
+        }
+
+        public static void ReverseOfNumber()
+        {
+            int number = 13456;
+            int reverseNumber = 0;
+            int remainder = 0;
+            while (number > 0)
+            {
+                remainder = number % 10;
+                reverseNumber = reverseNumber * 10;
+                reverseNumber = reverseNumber + remainder;
+                number = number /10;
+            }
+            Console.WriteLine(reverseNumber);
+        }
+
+        public static void PowOfNumber(int number, int n)
+        {
+            int count = 1;
+            int pow = number;
+            while (count < n)
+            {
+                pow = pow *number ;
+                count++;
+            }
+            Console.WriteLine($"{number}^{n} = {pow}");
+        }
+
+        public static void PropertyOfNumber()
+        {
+            Console.WriteLine("Enter number:");
+            int number = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("1. Whether the entered number is positive or negative: ");
+            Console.WriteLine("2. Is it simple");
+            Console.WriteLine("3. Is it divided into 2, 5, 3, 6, 9 without a residue");
+            Console.WriteLine("Choose the action:");
+            int action = Convert.ToInt32(Console.ReadLine());
+            switch (action)
+            {
+                case 1:
+                    if (number < 0)
+                    {
```

Comment:
No need to use brackets for single-line statement. Also you've already removed them in else.